### PR TITLE
Update first-interaction.yml

### PR DESCRIPTION
**Pull request by shuklaritvik06**
#### Description of the change

Fixed the bug in the first interaction workflow file

#### Checklist

**I agree to the following :-**

*   Added description of the change
*   I've read the [contributing guidelines](https://github.com/web3community/devprotocol.xyz/blob/main/CONTRIBUTING.md)
*   Search previous suggestions before making a new PR, as yours may be a duplicate.
*   I acknowledge that all my contributions will be made under the project's license.

Notes (**optional**): 

<a href="https://gitpod.io/#https://github.com/web3community/devprotocol.xyz/pull/155"></a>

### CODE REVIEW

File: .github/workflows/first-interaction.yml [hunk 1]
```diff
@@ -12,5 +12,5 @@ jobs:
...
+          issue-message: '<h1>Great to see you here!</h1> <br><br> Welcome to the community :tada: <br> Thank you for your valuable contributions! 🚀 We appreciate every and single contribution! 😄 <br><br> <h2>Join our amazing community 📢</h2> <br><br> Join our community to learn more about Blockchain, Web3, Dapps, and so much more 💻 <br><br> Here you can find all of our socials! 🐦 Share them with your friends and everyone you know. <br><br> - <a href="https://discord.gg/VwJp4KM">Discord server</a><br> - <a href="http://twitter.com/devprtcl">Twitter</a><br> - <a href="https://www.linkedin.com/company/dev-protocol">LinkedIn</a><br> - <a href="https://t.me/devprotocol">Telegram</a><br> - <a href="https://www.youtube.com/channel/UCN7m74tFgJJnoGL4zk6aJ6g">YouTube</a><br> - <a href="https://www.facebook.com/DevProtocol.xyz/">Facebook</a><br><br> We hope to see you there! 😁'
```
Comment (shuklaritvik06) [inline .github/workflows/first-interaction.yml:15]:
```suggestion
          issue-message: '<h1>Great to see you here!</h1> <br><br> Welcome to the community :tada: <br> Thank you for your valuable contributions! 🚀 We appreciate each and every contribution! 😄 <br><br> <h2>Join our amazing community 📢</h2> <br><br> Join our community to learn more about Blockchain, Web3, Dapps, and so much more 💻 <br><br> Here you can find all of our socials! 🐦 Share them with your friends and everyone you know. <br><br> - <a href="https://discord.gg/VwJp4KM">Discord server</a><br> - <a href="http://twitter.com/devprtcl">Twitter</a><br> - <a href="https://www.linkedin.com/company/dev-protocol">LinkedIn</a><br> - <a href="https://t.me/devprotocol">Telegram</a><br> - <a href="https://www.youtube.com/channel/UCN7m74tFgJJnoGL4zk6aJ6g">YouTube</a><br> - <a href="https://www.facebook.com/DevProtocol.xyz/">Facebook</a><br><br> We hope to see you there! 😁'
```

File: .github/workflows/first-interaction.yml [hunk 2]
```diff
@@ -12,5 +12,5 @@ jobs:
...
+          pr-message: '<h1>Great to see you here!</h1> <br><br> Welcome to the community :tada: <br> Thank you for your valuable contributions! 🚀 We appreciate every and single contribution! 😄 <br><br> <h2>Join our amazing community 📢</h2> <br><br> Join our community to learn more about Blockchain, Web3, Dapps, and so much more 💻 <br><br> Here you can find all of our socials! 🐦 Share them with your friends and everyone you know. <br><br> - <a href="https://discord.gg/VwJp4KM">Discord server</a> <br> - <a href="http://twitter.com/devprtcl">Twitter</a><br> - <a href="https://www.linkedin.com/company/dev-protocol">LinkedIn</a><br> -  <br> - https://t.me/devprotocol">Telegram</a><br> - a href="https://www.youtube.com/channel/UCN7m74tFgJJnoGL4zk6aJ6g">YouTube</a><br> <a href="https://www.facebook.com/DevProtocol.xyz/">Facebook</a>- <br><br> We hope to see you there! 😁'
```
Comment (shuklaritvik06) [inline .github/workflows/first-interaction.yml:16]:
```suggestion
          pr-message: '<h1>Great to see you here!</h1> <br><br> Welcome to the community :tada: <br> Thank you for your valuable contributions! 🚀 We appreciate each and every contribution! 😄 <br><br> <h2>Join our amazing community 📢</h2> <br><br> Join our community to learn more about Blockchain, Web3, Dapps, and so much more 💻 <br><br> Here you can find all of our socials! 🐦 Share them with your friends and everyone you know. <br><br> - <a href="https://discord.gg/VwJp4KM">Discord server</a> <br> - <a href="http://twitter.com/devprtcl">Twitter</a><br> - <a href="https://www.linkedin.com/company/dev-protocol">LinkedIn</a><br> -  <br> - https://t.me/devprotocol">Telegram</a><br> - a href="https://www.youtube.com/channel/UCN7m74tFgJJnoGL4zk6aJ6g">YouTube</a><br> <a href="https://www.facebook.com/DevProtocol.xyz/">Facebook</a>- <br><br> We hope to see you there! 😁'
```

File: .github/workflows/first-interaction.yml [hunk 3]
```diff
@@ -12,5 +12,5 @@ jobs:
...
+          pr-message: '<h1>Great to see you here!</h1> <br><br> Welcome to the community :tada: <br> Thank you for your valuable contributions! 🚀 We appreciate each and every contribution! 😄 <br><br> <h2>Join our amazing community 📢</h2> <br><br> Join our community to learn more about Blockchain, Web3, Dapps, and so much more 💻 <br><br> Here you can find all of our socials! 🐦 Share them with your friends and everyone you know. <br><br> - <a href="https://discord.gg/VwJp4KM">Discord server</a> <br> - <a href="http://twitter.com/devprtcl">Twitter</a><br> - <a href="https://www.linkedin.com/company/dev-protocol">LinkedIn</a><br> -  <br> - https://t.me/devprotocol">Telegram</a><br> - a href="https://www.youtube.com/channel/UCN7m74tFgJJnoGL4zk6aJ6g">YouTube</a><br> <a href="https://www.facebook.com/DevProtocol.xyz/">Facebook</a>- <br><br> We hope to see you there! 😁'
```
Comment (Panquesito7) [inline .github/workflows/first-interaction.yml:16]:
```suggestion
          issue-message: '<h1>Great to see you here!</h1> <br> Welcome to the community :tada: <br> Thank you for your valuable contributions! 🚀 We appreciate each and every contribution! 😄 <br> <h2>Join our amazing community 📢</h2> <br> Join our community to learn more about Blockchain, Web3, Dapps, and so much more 💻 <br><br> Here you can find all of our socials! 🐦 Share them with your friends and everyone you know. <br><br> - <a href="https://discord.gg/VwJp4KM">Discord server</a><br> - <a href="http://twitter.com/devprtcl">Twitter</a><br> - <a href="https://www.linkedin.com/company/dev-protocol">LinkedIn</a><br> - <a href="https://t.me/devprotocol">Telegram</a><br> - <a href="https://www.youtube.com/channel/UCN7m74tFgJJnoGL4zk6aJ6g">YouTube</a><br> - <a href="https://www.facebook.com/DevProtocol.xyz/">Facebook</a><br><br> We hope to see you there! 😁'
          pr-message: '<h1>Great to see you here!</h1> <br><br> Welcome to the community :tada: <br> Thank you for your valuable contributions! 🚀 We appreciate each and every contribution! 😄 <br><br> <h2>Join our amazing community 📢</h2> <br><br> Join our community to learn more about Blockchain, Web3, Dapps, and so much more 💻 <br><br> Here you can find all of our socials! 🐦 Share them with your friends and everyone you know. <br><br> - <a href="https://discord.gg/VwJp4KM">Discord server</a> <br> - <a href="http://twitter.com/devprtcl">Twitter</a><br> - <a href="https://www.linkedin.com/company/dev-protocol">LinkedIn</a><br> -  <br> - https://t.me/devprotocol">Telegram</a><br> - a href="https://www.youtube.com/channel/UCN7m74tFgJJnoGL4zk6aJ6g">YouTube</a><br> <a href="https://www.facebook.com/DevProtocol.xyz/">Facebook</a>- <br><br> We hope to see you there! 😁'
```

**Review by Panquesito7 — COMMENTED** [review]
I'm not sure if this will work properly as the `-` is intended to be used as a list. Something like this:

- Option 1
- Option 2
- Option 3

Will that work? Because it didn't work the previous time. 😅

**Comment by shuklaritvik06** [comment]
Yep it will not work like the markdown but as you can see above in the comment of GitHub bot it looks good, so I didn't removed that😅

**Review by Panquesito7 — APPROVED** [review]
LGTM! 🚀 Thank you for fixing it! 😄👍

**Review by krishguptadev — COMMENTED** [review]
LGTM!

**Pull request merged**